src/migrate_db_to_postgresql.py

```diff
@@ -202,9 +201,13 @@ def _retrieve_binaries(self, sanitized_dict, key):
                 logging.debug(f'Retrieving {analysis_key}')
                 tmp = self.sanitize_fs.get_last_version(sanitized_dict[key][analysis_key])
                 if tmp is not None:
-                    report = pickle.loads(tmp.read())
+                    try:
+                        report = pickle.loads(tmp.read())
+                    except ModuleNotFoundError:
```

Comment:
What causes this error? Why could there be a custom class?

Reply:
That is a valid question :sweat_smile: 
I tried it out with an old FACT instance and this exception occurred because it seems there was some kind of custom class stored somewhere in the analysis result (it seems we got rid of that in the meantime but catching the error if someone with an equally old FACT instance tries to migrate seems like a good idea)

Reply:
It could be from development or a custom plugin though (but IMHO catching that exception here would still be a good idea)

Reply:
Can you add a comment describing the cause of this Exception?